RadixWallet/Clients/AppsFlyerClient/AppsFlyerClient+Live.swift

```diff
@@ -0,0 +1,20 @@
+import AppsFlyerLib
+
+extension AppsFlyerClient: DependencyKey {
+	static var liveValue: AppsFlyerClient {
+		@Dependency(\.sensitiveInfoClient) var sensitiveInfoClient
+
+		return .init(
+			start: {
+				AppsFlyerLib.shared().appsFlyerDevKey = sensitiveInfoClient.read(.appsFlyerDevKey)
```

Comment:
We should probably not start the AppsFlyer if the necessary sensitive info is missing.
Applicable for CI, where we don't care to start and run AppsFlyer.